zanata-war/src/main/java/org/zanata/rest/editor/MediaTypes.java

```diff
@@ -32,6 +32,11 @@ public String toString() {
     public static final String APPLICATION_ZANATA_LOCALES_JSON =
             APPLICATION_ZANATA_LOCALES + JSON;
 
+    public static final String APPLICATION_ZANATA_SUGGESTIONS =
+            APPLICATION_VND_ZANATA + ".suggestions";
+    public static final String APPLICATION_ZANATA_SUGGESTIONS_JSON =
+            APPLICATION_ZANATA_SUGGESTIONS + JSON;
+
```

Comment:
I am not sure what's the benefit of using vnd mime type v.s. generic application/json or application/xml. To me, a specific vnd type will mean: if I am writing a client for it, I have to remember to set the Accept header to this type otherwise I will get 415 media not supported. I am not against it more of a question to ask. @seanf @carlosmunoz